geometry/nearby_points_sweeper.hpp

```diff
@@ -28,7 +29,8 @@ class NearbyPointsSweeper
   // Adds a new point (|x|, |y|) on the plane. |index| is used to
   // identify individual points, and will be reported for survived
   // points during the Sweep phase.
-  void Add(double x, double y, size_t index);
+  // Points with higher |priority| will be prefered at the Sweep phase.
```

Comment:
s/prefered/preferred/
s/at the Sweep phase/during the Sweep phase/

Reply:
done